logic/attestation_service.py

```diff
@@ -307,12 +313,29 @@ def get_airbnb_verification_code(eth_address, airbnbUserid):
         )
 
 
-def normalize_number(phone):
+def validate_airbnb_user_id(airbnbUserId):
+    if not re.compile(r"^\d*$").match(airbnbUserId):
+        raise ValidationError(
+            'AirbnbUserId should be a number.',
+            'airbnbUserId')
+
+
+def validate_email(email):
```

Comment:
The email is validated by the time it reaches this logic because it has gone through the [marshmallow schema](https://github.com/OriginProtocol/origin-bridge/blob/02ec0ef1f83afdc40973f47249445d0d3e5c0d59/api/modules/attestations.py#L33) which has built in validation for emails.

Reply:
Thanks for that will remove this reundant logic